### ISSUE TITLE
Payroll TypeError: can't multiply sequence by non-int of type 'float'

### ISSUE THREAD

**Issue by ccomb** [odoo/odoo#5154]
In the quantity field of a salary rule, replace '1.0' with '1,0':

```
  File ".../odoo/addons/hr_payroll/hr_payroll.py", line 605, in get_payslip_lines
    tot_rule = amount * qty * rate / 100.0
TypeError: can't multiply sequence by non-int of type 'float'
```

The error is very difficult to understand from the user's point of view...

**Comment by odony** [comment]
Just for the record, there's no need to open a separate issue when you're making a PR, the PR description is usually sufficient. A PR is a kind of issue, so we're saving half the housekeeping work :wink: Thanks!

**Comment by ccomb** [comment]
Ok sorry, it'll save me some time too...

**Comment by odony** [comment]
Fixed in 7.0 by merging #5155 at 34d45085358552794ee6303b387424ef1d409106, will be automatically forward-ported to later series. Thanks!
